src/lib/db/feature-tag-store.ts

```diff
@@ -102,11 +102,18 @@ class FeatureTagStore implements IFeatureTagStore {
         const stopTimer = this.timer('getAllForFeature');
         if (await this.featureExists(featureName)) {
             const rows = await this.db
-                .select(COLUMNS)
+                .select([...COLUMNS, 'tag_types.color as color'])
```

Comment:
what is the reason that color is not part of the COLUMNS?

Reply:
I'm not refactoring all endpoints to retrieve the tag color. Only two views currently need the tag color information. Therefore I'm limiting this iteration to only provide the joins with the tag type table and retrieve the color for these two specific endpoints.